### ISSUE TITLE
Error with null value

### ISSUE THREAD

**Issue by mankeomorakort** [igortrinidad/vue-number-format#2]
when input value it raise an error like below:
```

TypeError: Cannot read properties of null (reading 'toString')
    at isNegative (app.js:86881:50)
    at format (app.js:86809:20)
    at VueComponent.vueNumberFormat (app.js:86751:69)
    at app.js:77026:69
    at Proxy.renderList (app.js:92779:16)
    at fn (app.js:76989:55)
    at Object.normalized [as default] (app.js:92738:62)
    at VueComponent.genWrapper (app.js:127182:52)
    at Proxy.render (app.js:127189:31)
    at VueComponent.Vue._render (app.js:93698:22)
```

**Comment by igortrinidad** [comment]
Thanks for find this issue!
It was fixed on commit #https://github.com/igortrinidad/vue-number-format/commit/200a06393b14db2bcf3fe3f0cda559b688a7ed1a and its already available on the version 1.2.1 of the package, so just run `npm install vue-number-format` and you are good to use `null ` values on the v-model without any error...

**Comment by mankeomorakort** [comment]
Thanks
